base/setup/usetup/lang/ro-RO.h

```diff
@@ -1,4 +1,4 @@
-/* �tefan Fulea (stefan dot fulea at mail dot com) */
+/* Ştefan Fulea (stefan dot fulea at mail dot com) */
```

Comment:
What is changing there?

Reply:
I highly suspect the GitHub website forcibly changed the encoding type of the file to UTF-8. Looks like Katayama committed these patches in GitHub than using the command line prompt.

Reply:
I will revert file "base/setup/usetup/lang/ro-RO.h".

Reply:
For reminders, **NEVER EVER** use the GitHub website editor to manipulate localized resource files!

Reply:
Reverted. And then I will apply ro-RO.h partically...

Reply:
OK, done.